com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipelineGlobalSettings.Migration.cs

```diff
@@ -40,6 +41,16 @@ enum Version
             MigrationStep.New(Version.UpdateLensFlare, (HDRenderPipelineGlobalSettings data) =>
             {
                 FrameSettings.MigrateToLensFlare(ref data.m_RenderingPathDefaultCameraFrameSettings);
+            }),
+            MigrationStep.New(Version.MovedSupportRuntimeDebugDisplayToGlobalSettings, (HDRenderPipelineGlobalSettings data) =>
+            {
+#pragma warning disable 618 // Type or member is obsolete
+                var activePipeline = GraphicsSettings.currentRenderPipeline as HDRenderPipelineAsset;
```

Comment:
Please, update the upgrade guide of HDRP to mention this migration step as it will be destructive (i.e if some users have configure runtime debug display on other configuration but not on the one in graphics settings we will loose this information. It is ok as this settings is not really mean to be shipped or control like this but it is better to inform users in the upgrade guide (upgrading from 21.1 to 21.2 md file)

Reply:
Good, I'll add that. @RSlysz I wonder if the same thing should have been done for HDRP Asset -> HDRP Global Settings itself? Or is it not required because that migrates from `defaultRenderPipeline`, rather than from `currentRenderPipeline`? Should I also use `defaultRenderPipeline` here?